### ISSUE TITLE
Asking.

### ISSUE THREAD

**Issue by 12194916** [AnshulSood11/PPE-Detection-YOLO-Deep_SORT#4]
Hello sir, have you updated the code? Could you add some other labels like vest gloves so on. And Can I use my trained weights for certain labels and use it in your code? Please sir I need your help. Thanks.

**Comment by chen-ying-ying** [comment]
Hi, I wanted to run the full yolo but I can't get find them in the download link.

**Comment by AnshulSood11** [comment]
I have updated the links, can you try again?

**Comment by chen-ying-ying** [comment]
> I have updated the links, can you try again?
The file in the Training the model sections of the README file are still not available, and the links in the text are inaccessible. Can you send it to my email or put it directly in the zip archive of the entire project file?My email is 1197685786@qq.com
 The following three are the names of the files I want to get：
1、[backend.h5]

<img width="657" alt="1b7a279257d39fa7e3b0545fd8f0f1a0" src="https://github.com/AnshulSood11/PPE-Detection-YOLO-Deep_SORT/assets/153420593/3cf55b3e-35ea-4c99-9c15-694a1d9436e6">

**Comment by chen-ying-ying** [comment]
> I have updated the links, can you try again?
The three files in the quick start and Training the model sections of the README file are still not available, and the links in the text are inaccessible. Can you send it to my email or put it directly in the zip archive of the entire project file?My email is 1197685786@qq.com
 The following three are the names of the files I want to get：
1、[mars-small128.pb]
2、[full_yolo3_helmet_and_person.h5]
3、[backend.h5]
<img width="672" alt="e31aec38f5ca510f9ac8de402f211333" src="https://github.com/AnshulSood11/PPE-Detection-YOLO-Deep_SORT/assets/153420593/871ce927-d2eb-4dac-8342-bd3e4cb399fd">

<img width="657" alt="1b7a279257d39fa7e3b0545fd8f0f1a0" src="https://github.com/AnshulSood11/PPE-Detection-YOLO-Deep_SORT/assets/153420593/3cf55b3e-35ea-4c99-9c15-694a1d9436e6">

**Comment by chen-ying-ying** [comment]
The file in the Training the model sections of the README file are still not available, and the links in the text are inaccessible. Can you send it to my email or put it directly in the zip archive of the entire project file?My email ***@***.***
The following three are the names of the files I want to get：
1、[backend.h5]



&nbsp;


捂风挽笑
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Anshul ***@***.***&gt;; 
发送时间: 2024年5月24日(星期五) 中午12:51
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [AnshulSood11/PPE-Detection-YOLO-Deep_SORT] Asking. (Issue #4)














 
I have updated the links, can you try again?

**Comment by AnshulSood11** [comment]
What do you mean by inaccessible? What do you see on clicking the links? Are you using some VPN?

**Comment by chen-ying-ying** [comment]
> What do you mean by inaccessible? What do you see on clicking the links? Are you using some VPN?

Yes, used a VPN and after I kept refreshing it would download, probably because the page was unstable.Thanks for the heads up.

**Comment by chen-ying-ying** [comment]
<img width="960" alt="24aeb6e608505762bb926404fc10f5fa" src="https://github.com/AnshulSood11/PPE-Detection-YOLO-Deep_SORT/assets/153420593/db417f8f-65c0-430b-ba1c-a746f29632b3">
<img width="908" alt="8fa7eb7c72782fcc4afb5a5b95eb27b0" src="https://github.com/AnshulSood11/PPE-Detection-YOLO-Deep_SORT/assets/153420593/792dae7f-7356-4009-8802-f081714d374b">

But when I run the file predict.py, I get the error in the image above, it doesn't recognize it properly and all the data is 0.

**Comment by AnshulSood11** [comment]
Hope you are using a separate python virtual environment for this project and have installed the exact dependencies as in requirements.txt

**Comment by chen-ying-ying** [comment]
The virtual environment and dependencies are installed, and I don't know why I can't recognize them.
